docs/topics/multiplatform/multiplatform-direct-integration.md

```diff
@@ -0,0 +1,89 @@
+[//]: # (title: Direct integration)
+
+If you already have a Kotlin Multiplatform and an iOS project and want to develop them simultaneously to share code, you
+can set up direct integration through a special `embedAndSignAppleFrameworkForXcode` script.
+
+This script automates the process of connecting the Kotlin framework to iOS projects in Xcode. It's designed
+specifically for the Xcode environment.
+
+![Direct integration diagram](direct-integration-scheme.svg){width="700"}
+
+The idea of direct integration is to make the Kotlin build a part of the iOS build.
```

Comment:
This sentence seems to be illustrated by the diagram, so I'd put it before the image.

Overall, we introduce direct integration as a concept first, and then explain how `embedAndSignAppleFrameworkForXcode` is a part of it. So I'd combine "This script automates the process..." and "The script is added..." with the rest of the information below the image to visually and narratively separate the concept from the implementation.

Reply:
It also seems to me that "connecting the Kotlin framework to iOS projects" and "make the Kotlin build a part of the iOS build" are intersecting for the most part. Maybe, again, it would be clearer to use this abstract language only when explaining direct integration, and rely on "...the Kotlin artifact is built and included in
the derived data before running the iOS app build." to convey the role of `embedAndSignAppleFrameworkForXcode`.

Reply:
Thank you! I fixed the script vs gradle task confusion and ended up rearranging the whole intro:

- I looked closely and decided that the best sentence to describe the diagram is "The idea of direct integration is to make the Kotlin build a part of the iOS build.", so I put it before the image
- I also moved the repetitive abstract "makes the Kotlin build a part of the iOS build." to the overview